plugins/chat/lib/chat_sdk/message.rb

```diff
@@ -116,6 +116,7 @@ def create(
       enforce_membership: false,
       force_thread: false,
       strip_whitespaces: true,
+      blocks: nil,
```

Comment:
Minor not: would an empty array be a better default?

Reply:
the problem if I do that is that it will raise an error when you are not a bot as it will think you are trying to give blocks to the message when you can. I can check that blocks is empty, but then what's the point?

Reply:
I will merge it, but happy to hear more and do a followup if you have more feelings about this